src/views/FAQ.vue

```diff
@@ -5,7 +5,10 @@
         <h1>{{ $t("app.faq") }}</h1>
         <v-expansion-panels v-if="!loading" accordion>
           <v-expansion-panel v-for="(faq, i) in faqs" :key="i">
-            <v-expansion-panel-header>
+            <v-expansion-panel-header
+              :style="{ 'font-weight': getQuestionFontWeight(faq) }"
```

Comment:
Ich gebe zu, dass ich jetzt eine Weile raus bin, aber für mich sieht das auf den ersten Blick etwas kompliziert aus.

Alternative könnte das mit classes gelöst werden - das wäre konsistent zu den anderen (zugegeben wenigen) Stylings und mehr Vuetify-Style: https://vuetifyjs.com/en/styles/text-and-typography/#typography

Der Code wäre dann einfacher in der Art von
```
:class = getQuestionFontWeight(faq)
```
und die Funktion gibt dann `font-weight-bold` oder `font-weight-regular` zurück